workspace/welcome-to-xod/111-internet/patch.xodp

```diff
@@ -0,0 +1,171 @@
+{
+  "comments": [
+    {
+      "content": "Now you got a remote analog gauge for the data that you send to the cloud. You can place it on your desktop and watch for the moisture on your greenhouse in the countryside.\n\nAlso, another device might turn on or off the water pump, depending on the same data. Moreover, you can send a status of a water pump to the cloud and add some nodes here to indicate that the water pump is working.",
+      "id": "SyoB37C3I",
+      "position": {
+        "units": "slots",
+        "x": 10,
+        "y": 4
+      },
+      "size": {
+        "height": 2,
+        "units": "slots",
+        "width": 8
+      }
+    },
+    {
+      "content": "# Internet On Hardware\n\nYou might be familiar with the `tethering-inet` node and [XOD Cloud](https://xod.io/docs/guide/getting-started-with-feeds/) from previous chapters of the tutorial. In this chapter, we make an analog gauge for the data that you've sent before.\nHere, we placed nodes to update the value and check out how your servo will react for it.\n\n## Excercise\n\n1. Place the `xod-cloud/feeds/cloud-number` node to get the latest numeric value from the feed.\n2. Bind the feed name into a `FEED` input pin. The feed name should be the same as at the `cloud-number` node.\n3. Link `cloud-number` with `tethering-inet`, `servo`, and `clock` nodes.\n4. Bind a `PORT` to the `servo` node.\n5. Simulate the patch and look at how your servo is reacting. Update the cloud-stored value using the `tweak-number` on the right side of the program.\n\nNote that in the tutorial, we named a feed \"moisture\". Usually, the moisture is measured in percentages. So we expect values in a range from 0.00 to 1.00. If you use another range, place the `map` node before the `VAL` input of a `servo` node.",
```

Comment:
The patch should be ready-to-go in its initial form. An exercise should add functionality on top of that